server/service/integration_mdm_test.go

```diff
@@ -3418,25 +3393,31 @@ func (s *integrationMDMTestSuite) TestMDMAppleDiskEncryptionAggregate() {
 
 	// filtering by the "team_id" query param
 	generateAggregateValue(hosts[0:2], fleet.MDMOperationTypeInstall, &fleet.MDMDeliveryVerifying, true)
-	fvsResp = getMDMAppleFileVauleSummaryResponse{}
-	s.DoJSON("GET", "/api/latest/fleet/mdm/apple/filevault/summary", nil, http.StatusOK, &fvsResp, "team_id", strconv.Itoa(int(tm.ID)))
-	require.Equal(t, uint(2), fvsResp.Verifying)
-	require.Equal(t, uint(0), fvsResp.Verified)
-	require.Equal(t, uint(0), fvsResp.ActionRequired)
-	require.Equal(t, uint(0), fvsResp.Enforcing)
-	require.Equal(t, uint(0), fvsResp.Failed)
-	require.Equal(t, uint(0), fvsResp.RemovingEnforcement)
+
+	var expectedTeamDiskEncryptionSummary fleet.MDMDiskEncryptionSummary
+	expectedTeamDiskEncryptionSummary.Verifying.MacOS = 2
+	s.checkMDMDiskEncryptionSummaries(t, &tm.ID, expectedTeamDiskEncryptionSummary, true)
+
+	expectedNoTeamDiskEncryptionSummary.Verifying.MacOS = 0 // now 0 because hosts 1,2 were added to team 1
+	s.checkMDMDiskEncryptionSummaries(t, nil, expectedNoTeamDiskEncryptionSummary, true)
+
+	expectedTeamProfilesSummary := fleet.MDMProfilesSummary{Verifying: 2}
+	s.checkMDMProfilesSummaries(t, &tm.ID, expectedTeamProfilesSummary, &expectedTeamProfilesSummary)
+
+	expectedNoTeamProfilesSummary = fleet.MDMProfilesSummary{
+		Verifying: 0, // now 0 because hosts 1,2 were added to team 1
+		Pending:   6,
+		Failed:    2,
+	}
+	s.checkMDMProfilesSummaries(t, nil, expectedNoTeamProfilesSummary, &expectedNoTeamProfilesSummary)
 
 	// verified status for host 1
 	require.NoError(t, apple_mdm.VerifyHostMDMProfiles(ctx, s.ds, hosts[0], map[string]*fleet.HostMacOSProfile{prof.Identifier: {Identifier: prof.Identifier, DisplayName: prof.Name, InstallDate: time.Now()}}))
-	fvsResp = getMDMAppleFileVauleSummaryResponse{}
-	s.DoJSON("GET", "/api/latest/fleet/mdm/apple/filevault/summary", nil, http.StatusOK, &fvsResp, "team_id", strconv.Itoa(int(tm.ID)))
-	require.Equal(t, uint(2), fvsResp.Verifying)
-	require.Equal(t, uint(0), fvsResp.Verified)
-	require.Equal(t, uint(0), fvsResp.ActionRequired)
-	require.Equal(t, uint(0), fvsResp.Enforcing)
-	require.Equal(t, uint(0), fvsResp.Failed)
-	require.Equal(t, uint(0), fvsResp.RemovingEnforcement)
+	// TODO: Why is there no change to the verification status of host 1 reflected in the summaries?
```

Comment:
Looks like something that would need to be figured out before merging, or do you plan on addressing that in a subsequent PR?

Reply:
I plan on addressing it separately. I don't quite understand what the expected behavior is in this test. Whatever it is, it already exists on the old FileVault summary endpoint. I wanted to make a note here so we can take a closer look when we turn to the part of the story that deals with updating to verified status.